dask/dataframe/io/tests/test_parquet.py

```diff
@@ -196,6 +197,7 @@ def test_simple(tmpdir, write_engine, read_engine):
     ddf = dd.from_pandas(df, npartitions=2)
     ddf.to_parquet(fn, engine=write_engine)
     read_df = dd.read_parquet(fn, index=["a"], engine=read_engine)
+    read_df.compute(scheduler="synchronous")
```

Comment:
Curious why this was added -- internally `assert_eq` on the next line should call `.compute(scheduler="synchronous")`

Reply:
Opps - This was a temporary change for debugging.